src/client.js

```diff
@@ -33,21 +33,22 @@ var events = [
 	"whois"
 ];
 var inputs = [
+	// These inputs are sorted in order that is most likely to be used
+	"msg",
+	"whois",
+	"part",
 	"action",
+
 	"connect",
 	"invite",
 	"join",
 	"kick",
 	"mode",
-	"msg",
 	"nick",
 	"notice",
-	"part",
 	"quit",
 	"raw",
-	"services",
 	"topic",
-	"whois"
```

Comment:
I put whois at the top because it's triggered by clicking on a username. Ideally though, yes, we would have a direct hashmap of the commands.